src/views/PatientCardScreen.js

```diff
@@ -18,7 +18,7 @@ const PatientCardScreen = ({ navigation }) => {
         style={styles.button}
         title="test"
       >
-        <Text>Move to add patient screen</Text>
+        <Text>Move to patient list</Text>
```

Comment:
patients list

Reply:
To tylko taki placeholder, ale faktycznie jest błąd. Poprawione 👍